### ISSUE TITLE
[Feature] "Copying" LockMyPix' (many) features

### ISSUE THREAD

**Issue by BlastboomStrice** [leonlatsch/Photok#145]
Hello, I've been using LockMyPix for a while and I thought about goibg open source and using Photok instead. I'm probably asking too much and since I ~can't code, I'm ~just adding pressure to the developer(s). Anyways though, here we go:
There some things I've noticed which kinda prevent me from using the app.

The important ones are:
~~1)Inability to create folders amd organize my gallery.~~
2)No pattern lock (4 by 4 prefferably)
~~3)Inability to choose wether to delete imported media (preferably with an option to delete with a military grade deletion)~~
~~4)No shake-lock (shake to lock vault)~~
5)No dissapearing automatically from "recent apps" (like the apps you see when you press home button for long).

Less important yet very useful:
~~1)No double tap to move skip/rewind videos.~~
2)No copy/move/paste options
~~3)No fake vault/multiple vaults~~
~~4)No other file types to encrypt and no notes encryption~~
~~5)No taking pics from the app~~
6)No trash folder

Basically if you could copy LockMyPix' features I think it would be very nice.

Uhh, if I knew how to code I'd probably already be fixing these... Thanks for your work, probably the only open source private vault for android.👍👍

[Edit: crossed out points already discussed here]

**Comment by comparesheriff** [comment]
@BlastboomStrice
Hey, thanks for all the nice feature suggestions!
We already have a few of them in the backlog and the collections are currently being developed.
Here are the issues to the corresponding features of yours, I believe:

Important:
1. Folders/Collections - #135 
**2. Pattern Lock - No Issue Yet (although biometric unlock is another feature)**
3. Delete imported Media - #55 
4. Shake-Lock - #58 
**5. Disappearing from recent apps - No Issue Yet**

Less important:
1. More Features for Videos - #139 
**2. Copy/Move/Paste - No Issue Yet**
3. Fake Vault / Multiple Vaults - #83 
4. Other File Encryption - No Issue Yet (probably won't be done for now, as Photok was foremost meant to be for photos and videos)
5. Taking Pics in App - #87 
**6. Trash Folder - No Issue Yet**

Please do take a look, if the mentioned Issues are what you are expecting and comment your ideas if not already mentioned their. 
For features that are missing and have no issue yet, you can create an issue for each single feature. That way it is easier to get development started. For general ideas about stuff like you could also head over to the [discussions page](https://github.com/leonlatsch/Photok/discussions/categories/ideas).

Thanks again for your input :)

**Comment by BlastboomStrice** [comment]
Ohhh (and I tried checking before posting...). Thanks  for the extremely fast response.😅 I'm going to cross out the duplicate ones. Would you like a few screenshots from Lockmypix to take a look at their settings?

**Comment by leonlatsch** [comment]
As @comparesheriff mentioned. Please open seperate issues for the features.
Use the feature request template.

I will close this as a duplicate.
